web-api/src/persistence/postgres/cases/schema.ts

```diff
@@ -39,6 +39,7 @@ export const caseTableDefinition = {
   leadDocketNumber: DEFAULT as string | null | undefined,
   litigationCosts: DEFAULT as number | undefined,
   mailingDate: DEFAULT as string | undefined,
+  manuallyAddedToTrial: DEFAULT as boolean | undefined,
```

Comment:
I think we can remove this | undefined. Related to this, in the mapper `toKyselyNewCase`, I think we can do 

`
manuallyAddedToTrial: rawCase.manuallyAddedToTrial ?? false,
`